### ISSUE TITLE
Modeling various qualifiers in relation to creators

### ISSUE THREAD

**Issue by SamiNorling** [linked-art/linked.art#229]
As I'm working through modeling our data to Linked Art, I'm tracking issues in my Github repo, but I've got one now that I think is relevant for others: modeling qualifiers for creators. My issue and thoughts so far can be referenced here: https://github.com/IMAmuseum/LinkedArt/issues/9.

Essentially, we catalog links to Actors in a table on our Object records. The table links to our Actor entity, and provides some additional information about that Actors relationship to the object: Role and After/Follower. I'm tripping up a bit on the ways that the After/Follower values would be represented in the Linked Art model.

Of the values that we use in that field, I'm thinking that they can be split into discrete categories, which may be handled differently in the model:

These could be represented as unique Actor entities, connected to the Production Event via carried_out_by:
- Circle of
- Follower of (but minting a URI could be misleading, with multiple individual followers likely, though the data is not that specific)
- School of
- Workshop of

Qualifiers indicating confidence in attribution (if the Actors are associated with Production Event via carried_out_by, is there a way to state the degree of confidence in that assertion?):
- Attributed to
- Possibly
- Probably

Open for consideration:
- Copy of (Production Event wouldn't be carried_out_by the Actor linked in this row of data - this could be documented instead as a relationship between the Object that is a copy and the Actor that was copied)
- Formerly attributed to (Provenance event? - issue of whether time span is documented well in our data)

I'd appreciate any feedback/thoughts and/or pointers to where this may be addressed in the model! Curious to know how other museums are cataloging these as well. For example, if "Workshop of Titian" would be it's own Actor record, as opposed to this qualifier on a link to the real Titian record.

**Comment by beaudet** [comment]
NGA models these as separate actors, e.g. Rembrandt's Workshop is an entirely separate constituent from Rembrandt and is the primary artist of some art objects.  Rembrandt also has a role with those objects although his constituent record is secondary to the workshop's.  I'm by no means suggesting this is an optimal way to model such relationships (maybe it is maybe it isn't) but it's what we tend to do in such situations.  Related to this is the most prolific artist of all time, "anonymous", and her dozens of cousins, "anonymous 18th century", etc. etc.

**Comment by azaroth42** [comment]
Attribute Assignment possible pattern:
<img width="513" alt="attribute-assignment" src="https://user-images.githubusercontent.com/871868/71103677-5c30a600-216f-11ea-8d51-210f2e2c3805.png">

**Comment by azaroth42** [comment]
Style-of:
<img width="422" alt="attribute-style-of" src="https://user-images.githubusercontent.com/871868/71104431-a1090c80-2170-11ea-9ab3-e042bf569f16.png">

**Comment by johnmcquaid** [comment]
At the Frick Photoarchive we use several modifiers:
attributed to
circle of
copy of
follower of
forgery of
manner of (equivalent to style of in other institutions)
shop of (equivalent to workshop of/atelier)
and assistants
collaborator (used with two or more known artists)
alternative attribution
former attribution

two or more could also be used in conjunction with each other. Rembrandt, shop of, former attribution or Titian, copy of, alternative attribution.

We do not treat workshops as individual actors. They all link back to the person (this also has to do with our archaic classification system). There will be some issue as to what artists have a workshop as independent entity (without opening up the can of worms of someone like Van Dyck and his workshop). I don't know what problems may arise by using the same term "workshop" as a descriptor for an person as well as an independent corporate body.

**Comment by aisaac** [comment]
In the two diagrams above https://github.com/linked-art/linked.art/issues/229#issuecomment-567104458 and https://github.com/linked-art/linked.art/issues/229#issuecomment-567108391 the link to aat:possibly should be la:property_classified_as

**Comment by azaroth42** [comment]
Homework for everyone: 

* Find other situations when the attribute assignment pattern is needed to cover .1 properties
* List the attribution qualifiers in use in institutions

Aim -- determine whether future-proofing with a node that relates the .1 type is worth the complexity, compared to a small number of sub-properties of influenced_by (or other)

**Comment by ewg118** [comment]
We have an uncertainty model in Nomisma that might be replicable here: http://ecfn.fundmuenzen.eu/images/Tolle_Wigg-Wolf_Uncertainty.pdf (second to last slide). 

The model is a little bit old since the slideshow is from 2014, but I think it could be applicable to a component of the production event.

**Comment by azaroth42** [comment]
AAT Attribution Qualifiers:  https://www.getty.edu/vow/AATHierarchy?find=&logic=AND&note=&page=1&subjectid=300404264

**Comment by azaroth42** [comment]
Discussion at F2F:  https://docs.google.com/document/d/1jBLE5hDYOeGTS0s9e7MzwHgNNEfMjC3jJW8Pk6na26A/edit  (page 36)

**Comment by beaudet** [comment]
link to existing vocabularies that we have collected: https://docs.google.com/spreadsheets/d/1gyxb9Q31jvF0Zd_BZscqijA20u99FRgCJgDGmRMbPe0/edit#gid=1337687428

**Comment by Habennin** [comment]
and another topology

<img width="697" alt="Screen Shot 2020-08-12 at 6 30 19 PM" src="https://user-images.githubusercontent.com/12068255/90034982-f5e23000-dcc9-11ea-80af-8fdb96b522c6.png">

**Comment by azaroth42** [comment]
Decision on call of 2020-08-11 -- Go with the attribute assignment pattern as consistent with other uses of attribute assignment, and more faithful to the notions of "style of", "manner of", "circle of", "follower of", and so on where there isn't a Group with any sort of intention to act, just a set of people, one of which was the actor.

**Comment by azaroth42** [comment]
Notes: https://docs.google.com/document/d/125NgFRyJwJCoA2-o1MXpPMQiUA5tgqUzAds-QVif3LM/edit

**Comment by azaroth42** [comment]
Trying to maximize consistency at the (slight) expense of semantics, I think the best approach is (still) an Attribute Assignment, with the following patterns per facet in AAT:

* Attributions to a known creator:  An attribute assignment that assigns the creator to the production event, with the aat term in `classified_as` on the attribute assignment. (This is this model: https://github.com/linked-art/linked.art/issues/229#issuecomment-567104458)

* [unknown] Creator working with a known creator:  Attribute assignment that assigns a Group. The Formation of the Group was influenced by the creator. The type of group is given in `classified_as` on the Group (e.g. workshop, etc). The fudge here is that for "groups" like "pupils of" ... this is really a set of people, not a coherent Group that can take action. But the alternatives are all worse.

* [unknown] Creator not working directly with a known creator:  Same as above. "Pupils" are no more coherent than "followers" as a Group. 

* Creator was influenced by known creator:  Attribute assignment from the production to the artist, with assigned_property of influenced, and classified_as aat:manner / aat:style / etc.

All of them are from the production, to an actor. They're either assigning carried_out_by or influenced_by. We can double up the certainty classifications and the "type" classifications for "probably in the style of" or "possibly the workshop of".

**Comment by edwardanderson** [comment]
We have around 600 cases where our catalogue records a set of `XOR` attributions. For example, a glass beaker is recorded as being made _either_ in Amsterdam _or_ Haarlem. Sometimes the production events may attribute different persons, places and/or techniques to each of the possible production events. The set of possibilities may be more than two, but the intention is that only one of the possible events be true.

We can certainly use the [_Uncertain attributions_](https://linked.art/model/object/production/#uncertain-or-changing-attributions) pattern for these, and tag them all as "possibly". But I was wondering if we could usefully bundle the set of possible attributions? Our expectation is that this might make rendering the possibilities clearer on a website. And that it is closer to the cataloguer's intention of recording that one-of-_n_ options is possible, and not that any of _n_ might be true.

For example: `object/1` was made _either_ by `actor/1` in `place/1` with the technique `type/1` _or_ by `actor/2` in `place/2` with the technique `type/1`:

```turtle
<https://example.org/object/1> a crm:E22_Human-Made_Object ;
    crm:P108i_was_produced_by [
        a crm:E12_Production ;
        crm:P141i_was_assigned_by [
            a crm:E13_Attribute_Assignment ;
            # Wikidata: "exclusive or"
            crm:P2_has_type <http://www.wikidata.org/entity/Q498186> ;
            crm:P177_assigned_property_of_type crm:P9i_forms_part_of
            crm:P141_assigned [
                a crm:E12_Production ;
                crm:P14_carried_out_by <https://example.org/actor/1> ;
                crm:P32_used_general_technique <https://example.org/type/1> ;
                crm:P7_took_place_at <https://example.org/place/1> 
            ] , [
                crm:E12_Production ;
                crm:P14_carried_out_by <https://example.org/actor/2> ;
                crm:P32_used_general_technique <https://example.org/type/1> ;
                crm:P7_took_place_at <https://example.org/place/2> 
            ]
        ]
    ] .
```

I fished [XOR](http://www.wikidata.org/entity/Q498186) out of Wikidata as I didn't find an appropriate logical operator in the AAT.

But anyway. Is this a use-case that we should be trying to support with semantics or are we straying too far past an already-complex pattern and making the data too hard to parse?

**Comment by edwardanderson** [comment]
I've done a little more thinking on [this](https://github.com/linked-art/linked.art/issues/229#issuecomment-1856084109) and am coming around to having multiple `AttributeAssignment` instances that are members of a set classified as [mutually exclusive events](http://www.wikidata.org/entity/Q2625281) (did not find an AAT equivalent). 

So this would look like:

```turtle
:object-0
    crm:P108i_was_produced_by [
        a crm:E12_Production ;
        crm:P141i_was_assigned_by [
            a crm:E13_Attribute_Assignment ;
            crm:P177_assigned_property_of_type crm:P9i_forms_part_of ;
            la:member_of _:b0 ;
            crm:P141_assigned [
                a crm:E12_Production ;
                crm:P14_carried_out_by :actor-1 ;
                crm:P7_took_place_at :place-1
            ]
        ] ;
        crm:P141i_was_assigned_by [
            a crm:E13_Attribute_Assignment ;
            crm:P177_assigned_property_of_type crm:P9i_forms_part_of ;
            la:member_of _:b0 ;
            crm:P141_assigned [
                a crm:E12_Production ;
                crm:P14_carried_out_by :actor-2 ;
                crm:P7_took_place_at :place-2
            ]
        ]        
    ]
.

_:b0
    a la:Set ;
    rdfs:label "set-0" ;
    crm:P2_has_type <http://www.wikidata.org/entity/Q2625281>
.
```

This is still a bit tricky for consumers because the `Set` is a blank node and parsing the `_label` is undesirable. But overall feels like a more compatible pattern.

**Comment by beaudet** [comment]
It would be helpful to have a JSON-LD example of the proposed structure for stating that an artist was functioning as a painter when they were carrying out the production of a work.

**Comment by azaroth42** [comment]
Checked the documentation (production of objects and assertions) and the qualifiers are accounted for in the model and API. We could create recipes to enumerate all the possibilities, but the structure is there and documented.
